plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_remote_invoke.py

```diff
@@ -45,8 +45,8 @@ def launch_jobs(temp_dir: str) -> None:
             setup_globals()
             Singleton.set_state(singleton_state)
             HydraConfig.instance().set_config(sweep_config)
-            ray_init_cfg = HydraConfig.get().launcher.params.ray_init_cfg
-            ray_remote_cfg = HydraConfig.get().launcher.params.ray_remote_cfg
+            ray_init_cfg = HydraConfig.get().launcher.ray_init_cfg
+            ray_remote_cfg = HydraConfig.get().launcher.ray_remote_cfg
```

Comment:
Is this really the best way to pass in the ray configuration to this function?

Reply:
maybe? I'm not sure, would love suggestions on alternative.

Reply:
Is the ray cfg node here?
```python
sweep_config.hydra.launcher.ray_init_cfg
```